tests/src/test/java/alluxio/server/ft/journal/JournalShutdownIntegrationTest.java

```diff
@@ -75,7 +74,7 @@
       new ConfigurationRule(new ImmutableMap.Builder<PropertyKey, String>()
           .put(PropertyKey.MASTER_JOURNAL_TAILER_SHUTDOWN_QUIET_WAIT_TIME_MS, "100")
           .put(PropertyKey.MASTER_JOURNAL_CHECKPOINT_PERIOD_ENTRIES, "2")
-          .put(PropertyKey.MASTER_JOURNAL_LOG_SIZE_BYTES_MAX, "32")
+          .put(PropertyKey.MASTER_JOURNAL_LOG_SIZE_BYTES_MAX, "65")
```

Comment:
why is this change needed?

Reply:
Raft journal system requires that the journal loge size should be bigger than 64KB.

Reply:
can we make this 128 then? also, do you mean 64 bytes?

Reply:
Sorry, I mean 64 bytes, will change it to 128 bytes